src/commands/key2ds.rs

```diff
@@ -185,7 +185,7 @@ impl Key2ds {
             let rr = Record::new(owner, class, ttl, ds);
 
             if self.write_to_stdout {
-                writeln!(env.stdout(), "{}", rr.display_zonefile(DISPLAY_KIND));
+                writeln!(env.stdout(), "{}", rr.display_zonefile(DisplayKind::Simple));
```

Comment:
Note: Using `Simple` instead of `Tabbed` here leads to different output than the original `ldns-key2ds`.

Reply:
See existing PR https://github.com/NLnetLabs/dnst/pull/48.